tests/rptest/tests/e2e_shadow_indexing_test.py

```diff
@@ -1388,17 +1388,6 @@ def test_throttling(self, cloud_storage_type):
         self.logger.info("Start consumer")
         self.consume()
 
-        self.logger.info("Stop nodes")
-        for node in self.redpanda.nodes:
-            self.redpanda.stop_node(node, timeout=120)
-
-        self.logger.info("Start nodes")
-        for node in self.redpanda.nodes:
-            self.redpanda.start_node(node, timeout=120)
-
-        self.logger.info("Restart consumer")
-        self.consume()
```

Comment:
Makes sense. This makes me wonder why we restarted in the first place 🤔

Reply:
yeh not sure. probably copy/paste